pkg/planner/core/point_get_plan.go

```diff
@@ -998,6 +998,8 @@ func tryWhereIn2BatchPointGet(ctx sessionctx.Context, selStmt *ast.SelectStmt) *
 					break
 				}
 			}
+		} else if !tbl.IsCommonHandle && colName.Name.Name.L == model.ExtraHandleName.L {
+			handleCol = model.NewExtraHandleColInfo()
```

Comment:
Maybe we should update `whereColNames` here.

Reply:
> Maybe we should update `whereColNames` here.

whereColNames is searched within indices of table info, which doesn't (and shouldn't?) contain _tidb_rowid.
of coursenewBatchPointGetPlan could be modified to do so, but I think logically extra handle is a handle rather than an index.